src/main/java/hw_5/Task2/MyInt.java

```diff
@@ -0,0 +1,14 @@
+package hw_5.Task2;
+
+public class MyInt {
+    public int x;
```

Comment:
access modifiers are public?

Reply:
public int x here, because I need it to be accessible from outside the package